entrypoint.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 python3 -m flask db upgrade
-gunicorn --worker-tmp-dir /dev/shm -w 1 --worker-class=gthread --threads 4 -b 0.0.0.0:5666 --log-file /app/aguni.log --log-level info "arcsi:create_app('../config.py')"
+gunicorn --reload --worker-tmp-dir /dev/shm -w 1 --worker-class=gthread --threads 4 -b 0.0.0.0:5666 --log-file /app/aguni.log --log-level info "arcsi:create_app('../config.py')"
```

Comment:
This is for local work, or do you intend to patch the prod codebase as well? 

If needed for local work: We could look into a way to set this based on the runtime environment (local/dev/prod)

If this is intended, can you share the story?

Reply:
This is just for testing purposes. Since this is just a draft PR, it was more comfortable to push it, than stashing it in and out every time.